.mailmap

```diff
@@ -0,0 +1,24 @@
+Aleksandra Tarkowska <a.tarkowska@dundee.ac.uk> <A.Tarkowska@dundee.ac.uk>
+Anna Hamacher <anna.hamacher@hhu.de> <61734239+abhamacher@users.noreply.github.com>
+Carsten Fortmann-Grote <carsten.fortmann-grote@evolbio.mpg.de> <grotec@evolbio.mpg.de>
```

Comment:
Should not be listed

Reply:
Why not? Is this not the same contributor?

Reply:
discussed offline. Ignore